www/_layouts/docs.html

```diff
@@ -183,6 +183,21 @@ <h2 class="site-toc-title">{{ page.toc_text }}</h2>
 
         {% endif %}
 
+        <!-- Show warnings for special versions (i.e. non-latest, and dev) -->
+        {% if page.note_link and page.note_link_text %}
+
+            <div class="alert alert-warning docs-alert" role="alert">
+                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                {{ page.note }}
+                <a href="{{ page.note_link }}">
+                    {{ page.note_link_text }}
+                </a>
+            </div>
+
+        {% endif %}
```

Comment:
Please adjust this code with respect to the comment on `fetch_docs.js`, and to trigger when `plugin_version` is specified.